nengo_loihi/builder.py

```diff
@@ -31,6 +31,250 @@
 logger = logging.getLogger(__name__)
 
 
+class Interneurons(object):
+    """Defines parameters for a group of interneurons.
+
+    Interneurons are used on the chip to facilitate NEF-style connections,
+    where activities from a neural ensemble are first transformed into a
+    decoded value (which is stored in the activities and synapses of the
+    spiking interneurons), before being passed on to another ensemble
+    (via that ensemble's encoders).
+    """
+    def __init__(self, n, dt=0.001):
+        from nengo_loihi.neurons import LoihiSpikingRectifiedLinear
+        self.neuron_type = LoihiSpikingRectifiedLinear()
+
+        self.dt = dt
+
+        # firing rate of inter neurons
+        self._inter_rate = None
+
+        # number of inter neurons
+        self.n = n
```

Comment:
So `n` isn't actually the total number of neurons. It's the number of neurons per dimension per on/off pair (at least in all the interneuron types we have so far). This is because all our interneurons use a system of on/off pairs and separate interneurons for each dimension. Theoretically, you could have interneurons where it's just one big group of neurons, and each responds to multiple dimensions (just like an ensemble with multiple dimensions).

So we could call it `neurons_per_dimension`, but even that wouldn't be quite accurate because the actual neurons per dimension would be twice that (because we always have on/off pairs).

Reply:
Maybe `n_pairs` or `pairs_per_dimension`? I'm not okay with the one-character `n`.

Reply:
I'd also be okay with `size` I think